tests/ert/unit_tests/services/test_storage_service.py

```diff
@@ -192,3 +193,12 @@ def test_that_server_hosts_exists_as_san_in_certificate(change_to_tmpdir):
 
     assert set(sans) == set(hosts_from_urls)
     del os.environ["ERT_STORAGE_CONNECTION_STRING"]
+
+
+def test_that_an_exception_is_raised_if_a_storage_server_file_has_no_permissions(
+    change_to_tmpdir,
+):
+    Path("storage_server.json").write_text("{}", encoding="utf-8")
+    os.chmod("storage_server.json", 0o000)  # no permissions
```

Comment:
This might give pytest cleanup problems afterwards, so I think it is wise to make the file readable or delete it when the test is finished.

Reply:
Probably a good idea